src/libsync/theme.h

```diff
@@ -163,14 +163,6 @@ class OPENCLOUD_SYNC_EXPORT Theme : public QObject
      */
     virtual QUrl helpUrl() const;
 
-    /**
-     * Setting a value here will pre-define the server url.
-     *
-     * The respective UI controls will be disabled
-     *  setting $OPENCLOUD_OVERRIDE_SERVER_URL
-     */
-    virtual QUrl overrideServerUrl() const;
```

Comment:
As we now have the possibility to set this via the global settings, I saw no use in keeping it around